collect_app/src/test/java/org/odk/collect/android/widgets/items/SelectChoicesMapDataTest.kt

```diff
@@ -367,6 +367,32 @@ class SelectChoicesMapDataTest {
         assertThat(secondItem.largeIcon, equalTo(org.odk.collect.icons.R.drawable.ic_map_marker_with_hole_big))
     }
 
+    @Test
+    fun `entity system properties are filtered out`() {
```

Comment:
Not only entity system properties, but also `GEOMETRY`.

Reply:
That's checked in other tests (`additional children are returned as properties` for example). This test is specifically concerned with the entities properties.